use-timescale/compression/manual-compression.md

```diff
@@ -100,11 +100,20 @@ uncompressed chunks in your data, you can roll them up into a single compressed
 chunk.
 
 To roll up your uncompressed chunks into a compressed chunk, alter the compression
-settings to set the compress chunk time interval, and run compression operations
+settings to set the compress chunk time interval and run compression operations
 to roll up the chunks while compressing.
 
+<Highlight type="note">
+The default setting of `compress_orderby` is `'time ASC'` (the ascending or ASC command is used to sort the data returned in ascending order), which causes chunks to be re-compressed
```

Comment:
The default setting for `compress_orderby` is `time ASC`. You use `ASC` to sort the data returned in ascending order. This causes chunks to be re-compressed

Reply:
The default settings it `time DESC`. Changing to ASC avoids re-compression https://github.com/timescale/timescaledb/issues/5612#issuecomment-1525741498

Sorry, my bad writing..

Reply:
Not sure what you're asking? Do you want to replace the text with above?

Reply:
Your fix is awesome, thank you.